homework9/src/pages/Teachers/Teachers.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from "react";
+import { useSelector, useDispatch } from "react-redux";
+
+import MainTitle from "../../components/MainTitle";
+import TeacherCard from "../../components/TeacherCard";
+
+import { fetchTeachers } from "../../state/teachersSlice";
+
+import "./Teachers.css";
+
+export default function Teachers() {
+  const dispatch = useDispatch();
+  const teachers = useSelector((state) => state.teachers.data);
+  const status = useSelector((state) => state.teachers.status);
```

Comment:
const { data:teachers, status } = useSelector((state) => state.teachers);
instead of 13 and 14 lines